.github/workflows/e2e-test.yml

```
@@ -23,7 +23,7 @@ env:
   E2E_SLACK_TOKEN:       ${{ secrets.E2E_SLACK_TOKEN }}
   E2E_USE_LOCAL_SERVER:  false
   E2E_RESULT_FILEPATH:   'tests/e2e/results.json'
-  WC_L2_VERSION: '7.0.1'
+  WC_L2_VERSION: '7.1.0'
```

The newest WC `7.1` version is `7.1.1`

```suggestion
  WC_L2_VERSION: '7.1.1'
```